src/adapter/debugAdapter.ts

```diff
@@ -262,6 +264,48 @@ export class DebugAdapter {
     return {};
   }
 
+  async _toggleSkipFileStatus(params: Dap.ToggleSkipFileStatusParams): Promise<Dap.ToggleSkipFileStatusResult> {
+    if (params.resource) {
+      const absPath = urlUtils.fileUrlToAbsolutePath(params.resource);
+      if (absPath) {
+        params.resource = absPath;
+      }
+    }
+
+    if (!this.sourceContainer.scriptSkipper) {
+      if (!this._thread || !this._thread.cdp()) {
+        return {};
+      }
+      this.sourceContainer.initializeScriptSkipper(new ScriptSkipper([]));
+      this.sourceContainer.scriptSkipper!.setBlackboxSender(this._thread.cdp().Debugger);
+    }
+
```

Comment:
This requires `params.resource` but `_isSourceInCurrentStack` doesn't, should it work or not?